apps/condo/domains/organization/hooks/useCreateOrganizationModalForm.tsx

```diff
@@ -58,11 +59,15 @@ const prepareFinishFetchParams = ({ id, userId }) => ({
         },
     },
 })
-const prepareValidationErrorsMapping = ({ ValueIsTooShortMsg, TinTooShortMsg, TinValueIsInvalid }) => ({
-    [EMPTY_NAME_ERROR]: {
+const prepareValidationErrorsMapping = ({ ValueIsTooShortMsg, OrganizationNameIsInvalid, TinTooShortMsg, TinValueIsInvalid }) => ({
+    [NAME_TOO_SHORT_ERROR]: {
```

Comment:
also seems strange to use `[asd.asd.asd` notation here

Reply:
fixed